### ISSUE TITLE
Does not work with trailingSlash: true

### ISSUE THREAD

**Issue by matusfaro** [dkershner6/cdk-nextjs-export-s3-dynamic-routing#9]
This doesn't necessarily need a fix, but I stumbled upon this in my deployment: you can't use `trailingSlash: true` in your nextConfig as it's incompatible with the url rewriter CloudFront function that relies on static pages being as `a/b.html` rather than `a/b/index.html`.

**Comment by dkershner6** [comment]
This is true, I specifically avoided supporting this as building because it would actually use a significant amount more logic in the loop, causing possible cloudfront function timeouts (which was a major problem during beta versions) and shrink the number of pages allowed (due to code size).

Happy to be proven wrong here, if someone wants to give it a shot, but I am wary of success.
